frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineVisualizationTask.tsx

```diff
@@ -60,6 +63,22 @@ export const PipelineVisualizationTask: React.FC<PipelineVisualizationTaskProp>
       taskStatus.reason = runStatus.Cancelled;
     }
   }
+
+  const component = (
```

Comment:
Name is too generic. You could use a more descriptive name like `taskComponent` to be put inside Firehose.

Reply:
The variable is scoped to this component... not sure it really matters as name collision isn't going to be an issue. But I'll rename it to `taskComponent`.

Reply:
Done.